apps/testing-javascript/src/pages/api/skill/[...skillRecordings].ts

```diff
@@ -1,14 +1,24 @@
 import SkillRecordings, {
   type SkillRecordingsOptions,
+  defaultPaymentOptions,
+  StripeProvider,
 } from '@skillrecordings/skill-api'
 import {nextAuthOptions} from '../auth/[...nextauth]'
 
+const paymentOptions = defaultPaymentOptions({
+  stripeProvider: StripeProvider({
+    stripeSecretKey: process.env.STRIPE_SECRET_TOKEN,
+    apiVersion: '2020-08-27',
+  }),
+})
```

Comment:
as an example, this is TJS's skill-api endpoint configuring `PaymentOptions` with a `StripeProvider` instance which will give skill-api access to a configured Stripe client

if the Stripe client is present in the `PaymentOptions`, then stripe-based endpoints will work, otherwise they will 500.